test/horizon/client/default_test.exs

```diff
@@ -10,42 +12,58 @@ defmodule Stellar.Horizon.Client.CannedHTTPClient do
           body :: String.t(),
           options :: list()
         ) :: {:ok, map()} | {:error, map()}
```

Comment:
We should include all possible status codes that Horizon could return. For example, the 50x status.

Reply:
All possible statuses were added.